pyproject.toml

```
@@ -87,5 +87,10 @@ src_paths = "meltano/edk"
 known_first_party = ["meltano"]
 
 [build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.8", "poetry-dynamic-versioning"]
```

I like pinning the `build-system.requires` deps:

```suggestion
requires = [
    "poetry-core==1.5.0",
    "poetry-dynamic-versioning==0.21.3",
]
```

They're only ever installed into an isolated build environment, so they won't conflict with any package dependencies. With that concern out of the way, pinning them provides a nice amount of stability.

---

GitHub's `toml` syntax highlighting doesn't seem to like my suggestion. I'm not sure why.